pyne/tally.pyx

```diff
@@ -171,9 +174,102 @@ cdef class Tally:
         tal_name_bytes = tal_name.encode()
         self._inst = new cpp_tally.Tally(std_string(<char *> type_bytes), std_string(<char *> part_name_bytes), <int> ent, std_string(<char *> ent_type_bytes), std_string(<char *> ent_name_bytes), std_string(<char *> tal_name_bytes), <double> size, <double> norm)
 
+    
+    def _tally_tally_2(self, part_name, ent_geom, origin, i_mesh, j_mesh, k_mesh,
+            i_ints=[], j_ints=[], k_ints=[], e=[], e_ints=[], vec=[], axl=[], tal_name='', norm=1.0):
+        """Tally(self, type, part_name, ent, ent_type, ent_name, tal_name='', size=0.0, norm=1.0)
+         This method was overloaded in the C-based source. To overcome
+        this we will put the relevant docstring for each version below.
+        Each version will begin with a line of # characters.
+
+        ################################################################
+
+        Parameters
+        ----------
+        part_name : string
+        ent_geom : string
+        origin : (3d) list 
+        i_mesh : list
```

Comment:
Suggests to add the content type of these lists. For example, `i_mesh : list of float`, `i_ints : list of int (default: empty)`.